src/main/java/com/swapnixi/feedbackhub/controller/UserController.java

```diff
@@ -19,8 +21,22 @@ public class UserController {
 
     @PostMapping("/create")
     public ResponseEntity<User> createUser(@RequestBody User user) {
-        User createdUser = userService.createUser(user);
-        return ResponseEntity.status(HttpStatus.CREATED).body(createdUser);
+        try {
+            User createdUser = userService.createUser(user);
+            return ResponseEntity.status(HttpStatus.CREATED).body(createdUser);
+        } catch (UserAlreadyRegestredException e) {
+            throw new ResponseStatusException(HttpStatus.NOT_ACCEPTABLE, e.getMessage());
+        }
+    }
+
+    @GetMapping(path = "login/{email}/{password}")
+    public boolean userLogin(@PathVariable String email, @PathVariable String password) {
+        if (userService.loginUser(email, password)) {
+            ResponseEntity.accepted();
+            return true;
+        } else {
+            throw new ResponseStatusException(HttpStatus.UNAUTHORIZED, "Invalid Credentials");
+        }
```

Comment:
Users can login with valid credential's only, This API will take email and password as parameter.